pkg/volume/glusterfs/glusterfs.go

```diff
@@ -656,6 +659,12 @@ func parseClassParameters(params map[string]string, kubeClient clientset.Interfa
 			cfg.user = v
 		case "restuserkey":
 			cfg.userKey = v
+		case "gid":
+			reqGid, err = strconv.Atoi(v)
```

Comment:
is this gid required to be configured on the pod (in the api or by the kubelet) for it to access the data on the volume?